.github/workflows/format.yml

```diff
@@ -0,0 +1,17 @@
+name: Format
+
+on: [push, pull_request]
+jobs:
+  format:
+      name: Format
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Check formatting
+        run: |
+          pip install black
```

Comment:
I might pin black to a specific version - that can avoid an annoying edge case where if a new black version is released and reformats our code and breaks CI without us changing anything. I don't think we pin it in our setup.py file either which we may want to correct.

Reply:
I'm not sure we should pin it in the setup.py file since we're not calling on black within the fprime-tools module itself.